app/styles/components/_detailview.scss

```diff
@@ -74,7 +74,30 @@
       }
     }
 
-    /* put the default detail-block settings here */
+      /*because it is the only section with filter-tools within the header,
+        special handling/tweaking is required for the sessions detail section specifically */
+      &#sessions {
```

Comment:
#sessions is sufficient here, but we should avoid using id based selectors.  Also its likely we will use filter tools in the header for the program year list on the single program page.  We can cross the bridge when we come to it, but using a class based selector here would make it easier.